mobile/src/main/java/ch/epfl/sdp/mobile/ui/setting/SettingScreen.kt

```diff
@@ -75,7 +77,14 @@ fun SettingHeader(state: SettingScreenState, modifier: Modifier = Modifier) {
   ) {
     SettingPicture(state)
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
-      Text(state.name, style = MaterialTheme.typography.h5)
+      Row(horizontalArrangement = Arrangement.Center) {

```

Comment:
You may want to align the content vertically using `CenterVertically`.

Reply:
Good point